README.md

```diff
@@ -48,24 +71,82 @@ Then update to your own private key.
 
 ## Print wallet balances
 
-This script will print your wallet balances for each chain.
+To print your wallet balances for each chain, run:
 
 ```bash
 npm run check-balance [local|testnet]
 ```
 
-If not specified, this will print balances of the wallet for testnet.
+The aforementioned command pertains to specifying the intended environment for a project to execute on. It provides the option to choose between local and testnet environments by appending either `local` or `testnet` after the command. 
+
+An example of its usage is demonstrated as follows: `npm run check-balance local` or `npm run check-balance testnet`. It is noteworthy that the script defaults to testnet if no environment is explicitly specified.
 
 ## Running the local chains
 
+To start the local chains, run the following command:
+
 ```bash
 npm run start
 ```
-
-Leave this node running on a separate terminal before deploying and testing the dApps.
+**Note:** Leave this node running on a separate terminal before deploying and testing the dApps.
 
 ## Examples
 
--   [Evm Examples](/examples/evm/)
--   [Aptos Examples](/examples/aptos/)
--   [Web Examples](/examples-web/)
+This repository contains several examples of dApps that you can use as a starting point for building your own cross-chain applications. Each example is located in a subdirectory under the `/examples` directory. The following sections describe each example in more detail.
+
+- `evm`: This folder contains several basic and advanced examples of how to interact with Ethereum Virtual Machine (EVM) contracts.
+
+- `aptos`: This folder contains examples of how to use `Aptos` module to enable cross-chain functionality.
+
+- `web`: This folder contains examples of how to build web-based dApps that utilize cross-chain functionality.
```

Comment:
This isn't right, there isn't a `web` folder in `examples`.

Reply:
Updated